recipe/meta.yaml

```
@@ -23,15 +23,16 @@ requirements:
     - wheel
```

I think that setuptools is necessary for the build system
https://github.com/cherrypy/cherrypy/blob/89461873c230ac853da6154a4ec1aaaa723f938e/pyproject.toml#L4

```suggestion
    - wheel
    - setuptools >=45
```

---

It's already three lines above